plugin-server/src/main/graphile-worker/graphile-worker.ts

```diff
@@ -189,20 +212,23 @@ export class GraphileWorker {
         })
     }
 
-    async start(jobHandlers: TaskList): Promise<void> {
+    async start(jobHandlers: TaskList, crontab: CronItem[] = []): Promise<void> {
```

Comment:
non-blocking

Maybe missing an `info` here as with the other methods? Could be useful to log some info e.g. about the jobHandler names and crontabs that are being used.